dex-auth/dex-crds/base/config-map.yaml

```diff
@@ -18,6 +18,10 @@ data:
     oauth2:
       skipApprovalScreen: true
     enablePasswordDB: true
+    grpc:
+      # Cannot be the same address as an HTTP(S) service.
+      addr: 127.0.0.1:5557
```

Comment:
@swiftdiaries would it be better to expose this with `0.0.0.0:5557`?

The client service might need to connect to this dex service through this port and binding of `0.0.0.0` would allow the client service to connect across pods.

Reply:
Changed service to `0.0.0.0:5557`